collections/programming-languages/index.md

```diff
@@ -49,6 +49,7 @@ items:
 - livecode/livecode
 - coq/coq
 - vlang/v
+- dart-lang/sdk
```

Comment:
👋 @abraham Can you change this to `dart-lang/language`?

Reply:
@damaneice updated.

Reply:
Hi @damaneice can you elaborate on why you requested this change? 

dart-lang/language is a design repo for language designers; it's a pretty specific repo to link to. The main repo for the Dart language is /sdk, which is what most would be looking for.

This also seems inconsistent with other language, for example for go the link is to https://github.com/golang/go, not to https://github.com/golang/proposal, and Rust links to https://github.com/rust-lang/rust, not https://github.com/rust-lang/rfcs.